modules/nw-sriov-configuring-multiple-nodes.adoc

```diff
@@ -0,0 +1,154 @@
+
+In large clusters, draining nodes sequentially can be time-consuming, taking hours or even days. In time-sensitive environments, you can enable parallel node draining in an `SriovNetworkPoolConfig` custom resource (CR) for faster rollouts of SR-IOV network configurations. 
+
+To configure parallel draining, use the `SriovNetworkPoolConfig` CR to create a node pool. You can then add nodes to the pool and define the maximum number of nodes in the pool that the Operator can drain in parallel. With this approach, you can enable parallel draining for faster reconfiguration while ensuring you still have enough nodes remaining in the pool to handle any running workloads.
+
+[NOTE]
+====
+* A node can only belong to one SR-IOV network pool configuration. If a node is not part of a pool, it is added to a virtual, default, pool that is configured to drain one node at a time only.
+
+* The node might reboot during the draining process.
+====
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* You installed the SR-IOV Network Operator.
+* Nodes have hardware that support SR-IOV.
+
+.Procedure
+
+. Create a `SriovNetworkPoolConfig` resource:
+
+.. Create a YAML file that defines the `SriovNetworkPoolConfig` resource:
++
+.Example `sriov-nw-pool.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: SriovNetworkPoolConfig
+metadata:
+  name: pool-1 <1>
+  namespace: openshift-sriov-network-operator <2>
+spec:
+  maxUnavailable: 2 <3>
+  nodeSelector: <4>
+    matchLabels:
+      node-role.kubernetes.io/worker: ""
+----
+<1> The name for the custom resource object.
+<2> The namespace where the SR-IOV Network Operator is installed.
+<3> Specify an integer number, or percentage value, for nodes that can be unavailable in the pool during an update. For example, if you have 10 nodes and you set the maximum unavailable to 2, then only 2 nodes can be drained in parallel at any time, leaving 8 nodes for handling workloads.
+<4> Specify the nodes to be added to the pool.
+
+.. Create the `SriovNetworkPoolConfig` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-nw-pool.yaml
+----
+
+. Create a namespace:
++
+[source,terminal]
+----
+$ oc create namespace sriov-test
+----
+
+. Create a `SriovNetworkNodePolicy` resource:
+
+..  Create a YAML file that defines the `SriovNetworkNodePolicy` resource:
++
+.Example `sriov-node-policy.yaml` file
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov-nic-1
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  nicSelector:
+    pfNames: ["ens1"]
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+  numVfs: 5
+  priority: 99
+  resourceName: sriov_nic_1
+----
+
+.. Create the `SriovNetworkNodePolicy` resource by running the following command:
```

Comment:
I think the earlier instance uses indefinite because it hasn't previously existed. The definite article is used subsequently because it's the YAML resource we just created :man_shrugging: